pyfar/dsp/filter/audiofilter.py

```diff
@@ -4,6 +4,136 @@
 from . import _audiofilter as iir
 
 
+def allpass(signal, frequency, order, coefficients=None, sampling_rate=None):
+    """
+    Create and apply first or second order allpass filter.
+
+    Allpass filters have an almost constant group delay below their cut-off
+    frequency and are often used in analogue loudspeaker design.
+    The filter transfer function is based on Tietze et al. [#]_:
+
+    .. math:: A(s) = \\frac{1-\\frac{a_i}{\\omega_c} s+\\frac{b_i}
+                {\\omega_c^2} s^2}{1+\\frac{a_i}{\\omega_c} s
+                +\\frac{b_i}{\\omega_c^2} s^2},
+
+
+    where :math:`\\omega_c = 2 \\pi f_c` with the cut-off frequency :math:`f_c`
+    and :math:`s=\\sigma + \\mathrm{i} \\omega`.
```

Comment:
```suggestion
    .. math:: A(s) = \\frac{1-\\frac{a_i}{\\omega_c} s+\\frac{b_i}
                {\\omega_c^2} s^2}{1+\\frac{a_i}{\\omega_c} s
                +\\frac{b_i}{\\omega_c^2} s^2},


    where :math:`\\omega_c = 2 \\pi f_c` with the cut-off frequency :math:`f_c`
    and :math:`s=\\mathrm{i} \\omega`.
```
In our case we do not use the real part of the Laplace transform.